WorkspaceServer/Kernel/CSharpKernelExtensions.cs

```diff
@@ -68,22 +69,50 @@ public static CSharpKernel UseNugetDirective(this CSharpKernel kernel, INativeAs
                 {
                     Handler = async context =>
                     {
-                        var refs = await restoreContext.AddPackage(package.PackageName, package.PackageVersion);
+                        var message = $"Attempting to install package {package.PackageName}";
+                        if (!string.IsNullOrWhiteSpace(package.PackageVersion))
+                        {
+                            message += $", version {package.PackageVersion}";
+                        }
```

Comment:
Can we also get a notification about which actual version was installed in the case where the version was not specified?

Reply:
Will do this in a subsequent change